Applications/Resnet/tensorflow/resnet18.py

```diff
@@ -29,7 +29,7 @@
 # Fix the seeds across frameworks
 SEED = 412349
 random.seed(SEED)
-tf.compat.v1.set_random_seed(SEED)
+tf.random.set_seed(SEED)
```

Comment:
LGTM for this API.
But, please be careful when you replace APIs.

```
'tf.compat.v1.set_random_seed' is compatible with eager mode. However, in eager mode this API will set the global seed instead of the graph-level seed of the default graph. In TF2 this API is changed to [tf.random.set_seed](https://www.tensorflow.org/api_docs/python/tf/random/set_seed).
```